### ISSUE TITLE
bug(InteractivityChecker): Seeing a lot of warning in unit test 'Element matching ‘[cdkFocusInitial]'

### ISSUE THREAD

**Issue by sanjiv-saini** [angular/components#27389]
### Description

We are using a library modal component which uses cdkFocusInitial in one of its div element having tabindex set to -1. Modal component is being used in around ~20 of our components. While running unit test, we are getting a lot of warning 'Element matching ‘[cdkFocusInitial]'. I think isFocusable check of InteractivityChecker is failing as the modal is not open initially and is failing isVisible.

What is the need of showing this warning, There is no elegant way of suppressing this warning.

### Expected Behavior

Lots of warning with 'Element matching ‘[cdkFocusInitial]'

### Actual Behavior

Warning 'Element matching ‘[cdkFocusInitial]' should not be shown, or a better way should be provided to suppress the warning.

### Environment

- Angular: 15.2.2
- CDK/Material: 15.2.2
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by wagnermaciel** [comment]
@sanjiv-saini Could you provide a stackblitz example of how the cdkFocusInitial is being used?

**Comment by mmalerba** [comment]
Thanks for reporting this issue. I've closed it for now because it doesn't include enough information to easily reproduce the problem. As per our [contribution guidelines](https://github.com/angular/components/blob/main/CONTRIBUTING.md#-found-an-issue), please provide a link to a REPL reproduction (such as [StackBlitz](https://stackblitz.com/fork/components-issue)), or an explanation of why that is not possible, along with any other information needed to reproduce the issue.

If you could please open a new issue with the requested reproduction information, we will gladly investigate further.
